test/ode_solvers_test.cpp

```diff
@@ -0,0 +1,462 @@
+/*
+GeomScale Project
+
+Copyright (c) 2020
+  Vissarion Fisikopoulos
+  Apostolos Chalkis
+  Elias Tsigaridas
+  Marios Papachristou
+
+Contributed and/or modified by Marios Papachristou,
+as part of Google Summer of Code 2020 program.
+
+VolEsti is free software: you can redistribute it and/or modify it
+under the terms of the GNU Lesser General Public License as published by
+the Free Software Foundation, either version 3 of the License, or (at
+your option) any later version.
+
+VolEsti is distributed in the hope that it will be useful, but WITHOUT
+ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+for more details.
+
+See the file COPYING.LESSER for the text of the GNU Lesser General
+Public License.  If you did not receive this file along with HeaDDaCHe,
+see <http://www.gnu.org/licenses/>.
+*/
+
+#include "doctest.h"
+#include "Eigen/Eigen"
+#include "ode_solvers.h"
```

Comment:
please categorize include files as in other tests, i.e. first the external files with `<..>`
 and later the files of the library with `"..."`